### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by banzaman**
### Project requirements
-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -5,6 +5,29 @@ def index
...
+
+  def create
+    @current_user = current_user
+    @post = @current_user.posts.new(post_params)
+
+    if @post.save
+      redirect_to user_posts_path(@current_user), notice: 'Post created successfully.'
+    else
+      render :new
+    end
```
Comment (brainconnect93) [inline app/controllers/posts_controller.rb:25]: - [ ] Hello @banzaman, great effort working on your project, but I realized the form to add more posts to a particular user has not been achieved in your project. Kindly check the video recording below to have more insight on what I am talking about.

Video 👇 

[screen-capture (1).webm](https://github.com/banzaman/Blog-app/assets/78634841/4f1628e9-728f-489f-8c7c-c70874ae8209)

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @banzaman, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Git flow ✔️ 
- Blog app - add forms ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @banzaman, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Achieved ability to add posts for users :heavy_check_mark:
 
## [Optional] suggestions:
* [x]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by banzaman** [comment]
thank you for your time

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (banzaman) [reply app/controllers/posts_controller.rb:25]: I will work on that

**Pull request merged**